OmiseSDK/Resources/ja.lproj/Localizable.strings

```diff
@@ -1,13 +1,94 @@
+// Payment Methods
```

Comment:
Does this translation files need to checked by translators?

Reply:
I didn't add anything new there besides copy from Android SDK, so it's not really required for the scope of this task. But in general it would be great if we can do that.